--- /dev/null
+++ ../src/Resources/Pages/CreateRecord.php
@@ -172,8 +172,9 @@
     protected function handleRecordCreation(array $data): Model
     {
         $record = new ($this->getModel())($data);
+        $tenant = Filament::getTenant();
 
-        if ($tenant = Filament::getTenant()) {
+        if ($tenant && static::getResource()::shouldApplyTenantRelationship()) {
             return $this->associateRecordWithTenant($record, $tenant);
         }
 
